src/main/java/uk/ac/ebi/subs/fileupload/repository/repo/FileRepository.java

```diff
@@ -17,6 +17,6 @@
     File findByTusId(String tusId);
     Page<File> findBySubmissionId(String submissionId, Pageable pageable);
     File findByFilenameAndSubmissionId(String filename, String submissionId);
-    Page<File> findByUser(String User, Pageable pageable);
+    Page<File> findByCreatedBy(String User, Pageable pageable);
```

Comment:
What's the current use case for this query?

Reply:
There is no current use case for this. I thought we might want to implement a lookup later: uploaded files by given user. That's why I added an index to that field.

Reply:
Ok. I removed it and will put back if we would really need it